apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2024-10-10T03:56:07Z"
-  build_hash: 36c2d234498c2bc4f60773ab8df632af4067f43b
+  build_date: "2024-11-09T00:29:41Z"
+  build_hash: 9715a2a715317a76ae83825294ca50cde9afd97b
   go_version: go1.23.2
-  version: v0.39.1
-api_directory_checksum: 8882bcc0e344c5efe175e36d5a8e75bcaca88174
+  version: 9715a2a
```

Comment:
Can you please pull the code-gen tags (`git pull --tags --upstream`) and regenerate the controller?